livingdoc-engine/src/test/java/org/livingdoc/engine/execution/examples/scenarios/SelfCheckoutScenarioFixture.java

```diff
@@ -34,14 +31,14 @@ void checkout(Integer sum) {
 
     private static class SelfCheckout {
 
-        List<Pair<String, Integer>> items = new ArrayList<>();
+        Map<String, Integer> items = new TreeMap<>();
```

Comment:
even though this is just an example, but it should be possible to add the same item (name > String) multiple times if this is really a self checkout system :D